padd.sh

```diff
@@ -436,8 +438,8 @@ GetPiholeInformation() {
     ftl_heatmap=${green_text}
     ftl_check_box=${check_box_good}
     # Get FTL CPU and memory usage
-    ftl_cpu="$(ps -p "${ftlPID}" -o %cpu | tail -n1 | tr -d '[:space:]')"
-    ftl_mem_percentage="$(ps -p "${ftlPID}" -o %mem | tail -n1 | tr -d '[:space:]')"
+    ftl_cpu="$(ps h -p "${ftlPID}" -o %cpu | tr -d '[:space:]')"
+    ftl_mem_percentage="$(ps h -p "${ftlPID}" -o %mem | tr -d '[:space:]')"
```

Comment:
These line will generate an error if `getFTLPID()` returns `-1`. 

You need to change the test on [line 427](https://github.com/pi-hole/PADD/pull/313/files#diff-42c44861235d0baff7c03aea0fe6c7b7cb95f444c3a398c6d96da1290fc6bc01R427).

Reply:
Thanks. Fixed.